clients/pkg/promtail/positions/write_positions_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+package positions
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+func writePositionFile(filename string, positions map[string]string) error {
```

Comment:
An explanation of why this function is duplicated for Windows would be good for future reference.